src/Authentication/Controllers/SystemRegisterController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Platform.Authentication.Core.Models;
+using Altinn.Platform.Authentication.Core.SystemRegister.Models;
+using Altinn.Platform.Authentication.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Authentication.Controllers;
+
+/// <summary>
+/// CRUD API for SystemRegister
+/// </summary>
+[Route("authentication/api/v1/systemregister")]
+[ApiController]
+public class SystemRegisterController : ControllerBase
+{
+    private readonly ISystemRegisterService _systemRegisterService;
+
+    /// <summary>
+    /// Constructor
+    /// </summary>
+    /// <param name="systemRegisterService">The Service</param>
+    public SystemRegisterController(ISystemRegisterService systemRegisterService)
+    {
+        _systemRegisterService = systemRegisterService;
+    }
+
+    /// <summary>
+    /// Retrieves the List of all the Registered Systems, except those marked as deleted.
+    /// </summary>
+    /// <param name="cancellationToken">The Cancellation Token</param>
+    /// <returns></returns>
+    //[Authorize]
+    [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+    [HttpGet]
+    public async Task<ActionResult> GetListOfRegisteredSystems(CancellationToken cancellationToken = default)
+    {
+        List<RegisteredSystem> lista = [];
+
+        lista.AddRange(await _systemRegisterService.GetListRegSys(cancellationToken));
+
+        return Ok(lista);
+    }
+
+    /// <summary>
+    /// Retrieves a list of the predfined default rights for the Product type, if any
+    /// </summary>
+    /// <param name="productId">The Id of the Product </param>
+    /// <param name="cancellationToken">The cancellation token</param>
+    /// <returns></returns>
+    [HttpGet("product/{productId}")]
```

Comment:
i think it will be more clear and self explanatory if we can add defaultrights in the route before product f.eks defaultrights/product/{productId}